translations/de.json

```diff
@@ -0,0 +1,226 @@
+        "There was an error processing your request.": "Es gab einen Fehler dabei deine Anfrage zu bearbeiten.",  
+        "These values are used to determine craft/barter/price unlocks and flea market fees.": "Diese Werte werden dazu verwendet um Handwerksinformationen,Tauschgeschäfte, Preisinformationen sowie Flohmarktpreise freizuschalten",  
+        "Time": "Zeit",  
+        "Too many notes to display.": "zu viele Informationen für die Darstellung.",  
+        "Total": "Gesamt",  
+        "Trader restocks": "Händler Vorräte",  
+        "Traders": "Händler",  
+        "Unknown": "Unbekannt",  
+        "Value": "Wert",  
+        "Velo": "Velo",  
+        "Want to check the status of our services (api, website, bot, etc)?": "Willst du den Status unserer Services (api, website, bot, etc) wissen?",  
+        "Want to contribute to the bot or checkout the source code? View the project on GitHub!": "Willst du dich am Projekt beteiligen oder den Code ansehen? Besuch uns auf GitHub!",  
+        "Weight": "Gewicht",  
+        "Where [command] is one of the following commands:": "wo [command] einer der folgenden Befehle sein sollte:",  
+        "You can learn more about the bot's commands by entering:": "Du kannst mehr über die Befehle des Bots erfahren, indem du folgendes eintippst:",  
+        "You do not have any saved progress. Below are the defaults used to determine craft/barter/price unlocks and flea market fees.": "Du hast bisher keinen Fortschritt gespeichert, folgend erfährst die Standardwerte um Objekte freizuschalten sowie die Flohmarktgebühren.",  
+        "You have restock alerts enabled for": "Die Alarmierung neuer Vorräte ist aktiviert für",  
+        "You must be an administrator to set channel restock alerts.": "Du musst Administrator sein, um Alarmierungen für neue Vorräte konfigurieren zu können.",  
+        "You must invoke this command in the server with the channel in which you want restock alerts.": "Du musst diesem Befehl auf dem Server in dem Channel ausführen, der die Vorratsalarmierungen mitteilen soll.",  
+        "You must supply your [TarkovTracker API token](<https://tarkovtracker.io/settings/>) to link your account.": "Du musst deinen [TarkovTracker API token](<https://tarkovtracker.io/settings/>) angeben, um deinen Account zu verknüpfen.",  
+        "You need to specify a search term": "Du musst einen Suchbegriff definieren",  
+        "You need to specify an ammo type": "Du musst einen Munitionstyp definieren",  
+        "Your hideout progress will update from TarkovTracker {{updateTime}}.": "Dein Unterschlupf Fortschritt wird von TarkovTracker aktualisiert: {{updateTime}}."  
+    },  
+    "command": {  
+        "about": "Über",  
```

Comment:
In this "command" namespace, any value that doesn't end in "_desc" needs to be lowercase and with no spaces because they are command and option names.

Reply:
Fixed in [30d09b1](https://github.com/marvhh/stash/commit/30d09b12587a850c9e1e9dfc015f7a1b5cdad766)